tests/README.md

```diff
@@ -424,3 +424,13 @@ The tool will print some general statistics and it will give you more detailed i
 scenarios that yielded different results.
 Sometimes, different results may have been caused by different test environments, whereas sometimes
 the security content is different, and those scans can be identified by respective scanning dates only.
+
+## Known issues
+
+1 - Test suite fails to test rule with the message:
+
+```
+Rule <rule_id> has not been evaluated! Wrong profile selected in test scenario?
```

Comment:
Would it be possible to improve error message?

Reply:
I think it is. It would have to parse the oscap output and identify the error, but I think it is not a trivial change.